vickev/howdypix
# Pix 40 as a user i want to change the order of a picture to sort them by name or date so that i can see the picture i want first

## Description

Add Sorting the pictures by name and date.

To do this feature, I had to manipulate the structure of the database to include 2 new tables, and save every research into a new tuple, in order in the future to be able to do smart albums more easily.





## How to test?

- Remove any previous database
- Check-out this branch.
- Go to one album and sort the pictures.

## Checklist
